source/proof_document_mock/lib/document_mock_client.rb

```diff
@@ -0,0 +1,29 @@
+require 'proofer'
+
+module IdentityIdpFunctions
+  class DocumentMockClient < Proofer::Base
+    vendor_name 'DocumentMock'
+
+    required_attributes :encryption_key, :front_image_iv, :back_image_iv, :selfie_image_iv
+    required_attributes :front_image_url, :back_image_url, :selfie_image_url
+    required_attributes :liveness_checking_enabled
+
+    stage :document
+
+    UNVERIFIABLE_PHONE_NUMBER = '7035555555'.freeze
+    PROOFER_TIMEOUT_PHONE_NUMBER = '7035555888'.freeze
+    FAILED_TO_CONTACT_PHONE_NUMBER = '7035555999'.freeze
+
+    proof do |applicant, result|
+      plain_phone = applicant[:phone].gsub(/\D/, '').gsub(/\A1/, '')
+      if plain_phone == UNVERIFIABLE_PHONE_NUMBER
+        result.add_error(:phone, 'The phone number could not be verified.')
+      elsif plain_phone == FAILED_TO_CONTACT_PHONE_NUMBER
+        raise 'Failed to contact proofing vendor'
+      elsif plain_phone == PROOFER_TIMEOUT_PHONE_NUMBER
+        raise Proofer::TimeoutError, 'document mock timeout'
```

Comment:
If we're referenceing the `Proofer::` anything, then the mock funciton needs to depend on the `proofer` gem:

```
gem 'proofer', github: '18F/identity-proofer-gem', tag: 'v2.7.0'
```